website/templates/util/render_nodes.mako

```diff
@@ -1,6 +1,6 @@
 % if len(nodes):
     <ul class="list-group ${'sortable' if sortable and 'write' in user['permissions'] else ''}">
-      <span id="componentScope" class="scripted">
+      <span class="componentScope scripted">
```

Comment:
Our style guide says that classes and IDs follow different naming conventions. This class should be 'component-scope'.

I'd also strongly consider renaming it. `component-scope` isn't very description - perhaps `node-list-scope` or `rendered-nodes-scope`?